src/typescript/frontend/src/components/trade/DepositWithdrawModal/SelectMarketContent.tsx

```diff
@@ -1,116 +1,490 @@
+                      key={row.id}
+                    >
+                      {row.getVisibleCells().map((cell, i) => (
+                        <td
+                          className={
+                            i === 0
+                              ? "text-left text-white"
+                              : i === 6
+                              ? `${
+                                  cell.getValue() === "open" ? "text-green" : ""
+                                }`
+                              : ""
+                          }
+                          key={cell.id}
+                        >
+                          {flexRender(
+                            cell.column.columnDef.cell,
+                            cell.getContext()
+                          )}
+                        </td>
+                      ))}
+                    </tr>
+                  ))
+                )}
+              </tbody>
+            </table>
+          </div>
+          {/* <TableComponent table={table} /> */}
+        </Tab.Panels>
+      </Tab.Group>
+    </div>
+  );
+};
+
+// row components
+const MarketNameCell = ({ name }: { name: string }) => {
+  return (
+    <div className={`flex items-center text-base ${TABLE_SPACING.paddingLeft}`}>
+      <MarketIconPair
+        quoteAssetIcon={`/tokenImages/${name.split("-")[1]}.png`}
+        baseAssetIcon={`/tokenImages/${name.split("-")[0]}.png`}
+      />
+      <div className={`ml-7 min-w-[12em]`}>{name}</div>
+    </div>
+  );
+};
+
+const PriceCell = ({ price }: { price: number }) => {
+  const formatter = Intl.NumberFormat("en", {
+    notation: "compact",
+    compactDisplay: "short",
+    minimumFractionDigits: 1,
+    maximumFractionDigits: 1,
+  });
+  return (
+    <div>
+      <div className={`inline-block min-w-[8em] text-sm `}>
+        ${price >= 10_000 && formatter.format(price).replace("K", "k")}{" "}
+        {price < 10_000 &&
+          price.toLocaleString("en", {
+            minimumFractionDigits: 2,
+            maximumFractionDigits: 2,
+          })}
+      </div>
+      <div className={`inline-block min-w-[6em] text-neutral-500`}>$1.5M</div>
+    </div>
+  );
+};
+
+const VolumeCell = ({
+  volume,
+  baseAsset,
+}: {
+  volume: number;
+  baseAsset: string;
+}) => {
+  // is this ok? https://caniuse.com/mdn-javascript_builtins_intl_numberformat_numberformat_options_compactdisplay_parameter
+  // reference: https://stackoverflow.com/a/60988355
+  // also, people tend to use lower case 'k' but the formatter uses upper case 'K'
+  const formatter = Intl.NumberFormat("en", {
+    notation: "compact",
+    compactDisplay: "short",
+    minimumFractionDigits: 1,
+    maximumFractionDigits: 1,
+  });
+  return (
+    <div>
+      <div className={`inline-block min-w-[8em] text-sm`}>
+        {formatter.format(volume).replace("K", "k")} {baseAsset}
+      </div>
+      <div className={`inline-block min-w-[6em] text-neutral-500`}>$1.5M</div>
```

Comment:
I think volume will have to be base currency / quote currency. If we want to give volume in USD, there's a separate discussion to be had about where the USD price data comes from

Reply:
blocked: waiting for input from others